src/components/sbb-train/sbb-train.scss

```diff
@@ -0,0 +1,133 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-train-sector-padding: var(--sbb-spacing-fixed-1x);
+  --sbb-train-direction-spacing: var(--sbb-spacing-fixed-4x);
+  --sbb-train-direction-label-padding: var(--sbb-spacing-fixed-1x);
+  --sbb-train-direction-station-padding: var(--sbb-spacing-fixed-3x);
+  --sbb-train-direction-line-width: var(--sbb-border-width-1x);
+  --sbb-train-direction-line-color: var(--sbb-color-cement-default);
+  --sbb-train-direction-label-color: var(--sbb-color-smoke-default);
+  --sbb-train-direction-label-background: var(--sbb-color-white-default);
+
+  // Static value from design specs
+  --sbb-train-direction-indicator-end-height: #{px-to-rem-build(12)};
+
+  // Evalutate offset for the arrow-icon because 50% offset will cause a cut off of
+  // the arrow-top on the overflowing edge
+  --sbb-train-direction-indicator-arrow-offset-right: #{px-to-rem-build(9)};
+  --sbb-train-direction-indicator-arrow-offset-left: calc(
+    var(--sbb-train-direction-indicator-arrow-offset-right) * -1
+  );
+}
+
+.sbb-train {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  height: 100%;
+}
+
+.sbb-train__sectors {
+  display: flex;
+  gap: var(--sbb-train-sector-padding);
+}
+
+.sbb-train__direction {
+  padding-block-start: var(--sbb-train-direction-spacing);
+  margin-block-end: var(--sbb-train-direction-spacing);
+  display: flex;
+  position: relative;
+  margin-top: auto;
+  width: 100%;
+
+  h3 {
+    z-index: 1;
+    display: flex;
+    position: sticky;
+    inset: auto 0;
+    flex-direction: column;
+    width: 100%;
+    max-width: var(--sbb-train-direction-width);
+    margin: 0;
+    color: var(--sbb-train-direction-label-color);
+  }
+}
+
+.sbb-train__direction-indicator {
+  z-index: 0;
+  position: absolute;
+  top: calc(
+    ((var(--sbb-font-size-text-s) * var(--sbb-typo-line-height-body-text)) / 2) +
+      var(--sbb-train-direction-spacing)
+  );
```

Comment:
```suggestion
  top: calc(
    (0.5em) +
      var(--sbb-train-direction-spacing)
  );
```

Reply:
Sadly lead this not to the desired location